back-end/controllers/user.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const router = express.Router();
+const { queryDB } = require("../helpers/db-helpers");
+
+router.get("/user/:userId", async function (req, res, next) {
+  // will return {'username': 'sample', 'secret': 'sample} for given userid
+  try {
+    const userId = req.params["userId"];
+
+    const query = 
+      `SELECT username, secret 
+      FROM Users u INNER JOIN Secrets s 
+      ON u.user_id = s.user 
+      WHERE u.user_id = ${userId}`;
```

Comment:
👍 great query! Although 1 issue we might run into is that, although the user exists they might not have any secrets and this will return empty data.